bookkeeper-common/src/main/java/org/apache/bookkeeper/common/collections/RecyclableArrayList.java

```diff
@@ -64,4 +64,15 @@ public void recycle() {
             handle.recycle(this);
         }
     }
+
+    @Override
+    public boolean equals(Object obj) {
+        return super.equals(obj);
+    }
+
+    @Override
+    public int hashCode() {
```

Comment:
why do we need this? if it is needed, add a comment.

Reply:
Spot bugs is not happy